search_service/api/table.py

```diff
@@ -7,6 +7,11 @@
 
 from search_service.proxy import get_proxy_client
 
+badge_fields = {
+    "tag_name": fields.String,
+    "tag_type": fields.String
+}
+
```

Comment:
Mmm... Are we using flask_restful schema here? I can see it's declaring to use both `marshal_with` and `swagger_doc` where we are all moving toward to swagger doc.
https://github.com/lyft/amundsensearchlibrary/pull/85/files#diff-c32567583fb6dd003112a45fec024d24R57-R58

@csteez Could you try out with only swagger by removing line 57 and by updating swagger yml file?
swagger_doc/table/search_table.yml